packages/Graph/Actions/User/GetUserProfile.cs

```diff
@@ -50,9 +62,18 @@ protected override void PopulateParameters(DialogStateManager state, Dictionary<
                 throw new ArgumentNullException(nameof(this.UserId));
             }
 
+            // Select minimum of the "id" field from the object
+            string fieldsToSelect = DefaultIdField;
```

Comment:
This logic is being repeated in every single action. Is that really necessary? I don't know that we need to fix that this round, but this definitely feels like we have an opportunity to reduce code duplication.

Reply:
Not necessarily every action - only ones that have to do with User objects. The duplication here is not too bad so I don't feel there is pressing need to refactor them.